index.html

```diff
@@ -190,15 +190,18 @@ <h2 id="docconformance">
         MUST NOT overwrite the implicit ARIA semantics, or native semantics
         of the HTML element.
       </p>
-
       <p class="note" id="aria-usage-note">
         While setting an ARIA `role` and/or `aria-*` attribute that matches the
         <span>implicit ARIA semantics</span> is NOT RECOMMENDED, in some
         situations explicitly setting these attributes can be helpful.
         For instance, in user agents which lack exposing specific implicit
         ARIA semantics.
       </p>
-
+      <p>
+        The values of ARIA `role` and/or `aria-*` attribute are case insensitive.
```

Comment:
It should mention if the attributes use Unicode case-insensitive or ASCII case-insensitive matching. See https://www.w3.org/TR/charmod-norm/#handlingCaseFolding for more info.